monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -99,6 +99,20 @@
                 }
             ]
         },
+        "system_info_collectors_classes": {
```

Comment:
I'd rather we hide this from UI. It raises a lot more questions than it solves.

Reply:
As we talked on the call, we'll keep this for now but we'll have to do a cleanup of the config and move A LOT to "expert mode" soon.

Reply:
https://github.com/guardicore/monkey/issues/526

Reply:
My arguments for hiding this in the following release:
1. I think turning system info collection on/off is enough control for most users.
2. It looks like we only have a single info collector, which is not true